powersimdata/input/change_table.py

```diff
@@ -567,6 +568,10 @@ def add_demand_flexibility(self, info):
                     del self.ct["demand_flexibility"]
                     raise ValueError(f"Available {k} profiles: {', '.join(possible)}")
 
+    def add_electrification(self, kind, info):
+        """TODO"""
```

Comment:
How do we want to do docstrings? Currently for example, in `scale_renewable_stubs` we are using (3). 
1. both in `change_table.py` and the imported function
2. only in `change_table.py`, and put a link in the docstring for the imported function 
3. only in the imported function, and put a link to that module here
4. something else?

Reply:
I like (3) the best, followed by (1).

Reply:
I think we either do both (1) or only in the function that being called first (I guess this is equivalent to 3?).

Reply:
I like 3. This is what you did in #604